apis/generators/v1alpha1/types_generator_state.go

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Call this file `generatestate_types.go` so its easier to distinguish between types of generator which are all prefixed with `types_xxx.go`.